test/compileStyle.spec.ts

```diff
@@ -57,7 +57,7 @@ test('preprocess sass', () => {
   const style = parse({
     source:
       '<style lang="sass">\n' +
-      '$red: rgb(255, 0, 0);\n' +
+      '$red: rgb(255, 0, 0)\n' +
```

Comment:
It would be a breaking change IMO. Though dart-sass & node-sass are both compliant with the sass  spec, there're many subtle differences underlyingly, especially when it comes to error tolerance.

Reply:
Is it worth the breaking change for not be needing to compile `node-sass`?

Reply:
It won't affect vue-loader but rollup-plugin-vue would need a major bump /cc @znck

Reply:
Yes, it would be a major bump for compiler utils as well as RPV.

Reply:
So, let's leave it up to you to decide when to merge this PR and release a new major for both libraries 👀